iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ShuntCompensatorImpl.java

```diff
@@ -275,4 +310,11 @@ public void allocateVariantArrayElement(int[] indexes, final int sourceIndex) {
     protected String getTypeDescription() {
         return "Shunt compensator";
     }
+
+    private Integer checkSolvedSectionCount(Integer solvedSectionCount, int maximumSectionCount) {
+        if (solvedSectionCount != null && (solvedSectionCount < 0 || solvedSectionCount > maximumSectionCount)) {
```

Comment:
There is an inconsistency: the javadoc of `getSolvedSectionCount()` says that `solvedSectionCount` should be greater than 1.
Note that it is also the case for `sectionCount` which can be set to 0 (see `setSectionCount()`) although `getSectionCount()'s javadoc says it should be greater than 1 too.

Reply:
I guess the javadoc is incorrect but I am not sure

Reply:
@colinepiloquet: can you also update the Javadoc of `getSectionCount()`?